ivy/functional/frontends/torch/comparison_ops.py

```diff
@@ -1,4 +1,9 @@
-# ToDo: Add allclose(), isclose(), isposinf(), isneginf(), fmax() to functional API
+"""
+ToDo
+----
+Add allclose(), isclose(), isposinf(), isneginf(), fmax(), msort()
```

Comment:
I think if we are only using 1 ivy function then its not worth adding to the API

Reply:
Yeah it makes sense.